src/fruity/lockdown.vala

```diff
@@ -77,15 +77,11 @@ namespace Frida.Fruity {
 
 		public async IOStream start_service (string name_with_options, Cancellable? cancellable = null) throws LockdownError, IOError {
 			bool tls_handshake_only = false;

```

Comment:
How about moving this right after `name`, as it's lower level in a sense?

Reply:
ah yeah, makes sense. i've put it at the beginning because it's like the "outcome" of the whole block of code.